src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure implements Drawable, AreaCalculable {
+    private double sideA;
+    private double sideB;
+
+    public Rectangle(double sideA, double sideB, String color) {
+        super.setColor(color);
+        this.sideA = sideA;
+        this.sideB = sideB;
+    }
+
+    @Override
+    public double area() {
```

Comment:
The method name 'area' is a verb, which is not recommended for method names in interfaces. Consider using a noun or a noun phrase, for example, 'calculateArea' would be more appropriate.